app/models/concerns/job_posting_searchable.rb

```diff
@@ -20,9 +19,11 @@ module JobPostingSearchable
         Rails.application.routes.url_helpers.job_posting_url(self, host: ENV['APP_HOST'], port: ENV['APP_PORT'])
       end
 
-      attribute :relevancy do
-        relevancy_for_database
-      end
+      attribute(:relevancy) { relevancy_for_database }
+
+      attribute(:description) { description.body&.to_plain_text }
```

Comment:
Can `description` maybe also return nil? So that calling `.body` would not work?

Reply:
Good question

Reply:
But I believe no. I checked a job posting with a known empty rich text and algolia index shows a null